### ISSUE TITLE
block issues

### ISSUE THREAD

**Issue by nihui** [Tencent/ncnn#3700]
~~thread affinity on offline core (https://github.com/Tencent/ncnn/issues/3273)~~
- [x] vs2022 package
- [x] reflect border type
~~small pnnx binary~~
- [x] update use-ncnn-with-own-project doc
- [x] gap fp32 accumulator (https://github.com/Tencent/ncnn/issues/3561)
~~batchnorm fp32 accumulator (https://github.com/Tencent/ncnn/issues/3607)~~
- [x] pnnx save ncnn fp16 bin
- [x] optimize softmax arm (https://github.com/Tencent/ncnn/issues/3670)
- [x] test new moltenvk (https://github.com/Tencent/ncnn/issues/3698)
- [x] mips msa use no signal comparison (https://github.com/Tencent/ncnn/issues/3430)
~~update doc for rtti and exception~~
- [x] fix armhf toolchain build (https://github.com/Tencent/ncnn/issues/3704)
- [x] print Mat content doc
